pages/templates/_slug.vue

```diff
@@ -89,11 +86,14 @@ export default {
       const metaData = {
         type: 'website',
         url: `https://formester.com/templates/${this.$route.params.slug}/`,
-        title: this.template?.name || 'Formester',
-        description: this.template?.description || this.template?.name,
-        mainImage: this.template?.preview_image_url
-          ? this.template.preview_image_url
-          : 'https://formester.com/formester-form-builder-background.png',
+        title: this.template?.metaTitle || this.template?.name || 'Form Template | Formester',
+        description:
+          this.template?.metaDescription ||
+          this.template?.description ||
```

Comment:
Can we destructure this.template like this?
  const {
    metaTitle,
    name,
    metaDescription,
    description,
    previewImageUrl
  } = this.template;
  
  and use it in the meta data like: 
  title: metaTitle || name || 'Form Template | Formester',